src/WSRequestHandler.cpp

```diff
@@ -43,6 +43,9 @@ const QHash<QString, RpcMethodHandler> WSRequestHandler::messageMap {
 
 	{ "BroadcastCustomMessage", &WSRequestHandler::BroadcastCustomMessage },
 
+	{ "ProcessHotkeyByName", &WSRequestHandler::ProcessHotkeyByName },
```

Comment:
I think `TriggerHotkeyByName` and `TriggerHotkeyBySequence` would be better names, as they fall more in line with how the OBS api references them.

Reply:
Not so sure about the "sequence" part. OBS refers to that as key combination. But anyway, I'm fine with whatever you prefer

Reply:
I meant as in the `Trigger` language